test/clj/centrebull/test/shooters/core.clj

```diff
@@ -18,4 +18,16 @@
       (with-redefs [dao/create! (mock-dao/create! expected nil)]
         (let [{:keys [status body]} (shooters/create! {:body-params expected})]
           (is (= body nil))
-          (is (= status 200)))))))
\ No newline at end of file
+          (is (= status 200))))))
+
+  (testing "shooter-suggest"
+    (let [rv (gen-shooter) es "Johnny Search Term"]
```

Comment:
Rename `rv` to be `expected` and have the test `(is (= body expected))`

(gen-shooter) will have some randomization shortly, which will cause this test to break otherwise.